### PR TITLE
Feature/87 candidate selection

### PR DESCRIPTION
**Pull request by rcffc**

### CODE REVIEW

File: inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingService.java [hunk 1]
```diff
@@ -94,6 +95,9 @@
     private static final String POS_NOUN_PREFIX = "N";
     private static final String POS_ADJECTIVE_PREFIX = "J";
 
+    private Map<String, Set<CandidateEntity>> candidateCache =
+        new Collections.synchronizedMap(LRUCache<>(1024));
```
Comment (reckart) [inline inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingService.java]:
```
    private Map<String, Set<CandidateEntity>> candidateCache = Collections
            .synchronizedMap(new LRUCache<>(1024));
```

Comment (rcffc) [reply inception-concept-linking/src/main/java/de/tudarmstadt/ukp/inception/conceptlinking/service/ConceptLinkingService.java]: Thanks!

**Comment by reckart** [comment]
Dependencies need fixing:

```
[INFO] --- maven-dependency-plugin:3.0.2:analyze-only (default) @ inception-concept-linking ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.springframework.boot:spring-boot:jar:1.5.10.RELEASE:compile
[WARNING]    org.apache.wicket:wicket-spring:jar:7.10.0:compile
```

**Comment by reckart** [comment]
@rcffc can you please resolve the conflict?

**Pull request merged**